library/core/src/num/int_sqrt.rs

```diff
@@ -0,0 +1,190 @@
+/// These functions compute the integer square root of their type, assuming
+/// that someone has already checked that the value is nonnegative.
+
+const ISQRT_AND_REMAINDER_8_BIT: [(u8, u8); 256] = {
```

Comment:
Please add a doc comment

Reply:
OK, I've changed it to (but haven't yet force pushed):

```
//! These functions compute the integer square root of their type, assuming
//! that someone has already checked that the value is nonnegative.

/// This array stores the integer square roots and remainders of each index.
/// For example, `ISQRT_AND_REMAINDER_8_BIT[17]` will be `(4, 1)` because the
/// integer square root of 17 is 4 and because 17 is 1 higher than 4 squared.
const ISQRT_AND_REMAINDER_8_BIT: [(u8, u8); 256] = {
```

Reply:
Above and beyond :)